MyRecipes/ViewModels/RecipesViewModel.swift

```diff
@@ -9,6 +9,17 @@ import Foundation
 
 class RecipesViewModel: ObservableObject {
     @Published private(set) var recipes: [Recipe] = []
+    @Published var filteredRecipe = [Recipe]()
+    @Published var searchText = ""
+    
+    var filteredRecipes: [Recipe] {
+        guard !searchText.isEmpty else { return recipes }
+        
+        return recipes.filter { $0.name.localizedCaseInsensitiveContains(searchText) }
+//        return recipes.filter { recipe in
```

Comment:
is there a reason for keeping the commented code here?

Reply:
done